app/functions.php

```diff
@@ -8,6 +8,53 @@
 *
 */
 
+/* ###### Get all posts ###### */
+
+function get_posts(){
+	
+	// Get Firebase JSON
+	$json = file_get_contents(FIREBASE_URL.'/posts.json');
```

Comment:
Yeah, so it would look something like this:
`$json = file_get_contents(FIREBASE_URL.'/posts.json?orderBy="status"&equalTo='.$params->status);`
if it was set, obviously.

Would just reduce the initial overhead by returning a smaller JSON file